src/modules/course/dto/create-course.dto.ts

```diff
@@ -40,4 +41,21 @@ export class CreateCourseDto {
   @IsNotEmpty()
   @IsArray()
   mentor: [];
+
+  @IsNotEmpty()
+  tags: TagType;
+
+  @IsNotEmpty()
+  courseDetails: string;
+
+  @IsNotEmpty()
+  courseLevel: string;
```

Comment:
It should be enum
`Beginner`, `Intermediate`, `Training` and `advanced`

Reply:
Ok will do